gm4_mountaineering/data/gm4_mountaineering/functions/climbing_main.mcfunction

```diff
@@ -0,0 +1,9 @@
+# Running on a faster clock to enable smoother climbing
+
+# Adds vertical climbing abilities when wearing crampons and not holding anything
+execute as @a[gamemode=!spectator,tag=gm4_mountaineering_using_crampons] at @s run function gm4_mountaineering:climbing_effects/crampons_equipped
+
+# Apply damage to any climbing shulkers
+execute as @e[type=minecraft:shulker,tag=gm4_mountaineering_climb_assist] at @s run function gm4_mountaineering:climbing_effects/damage_shulkers
```

Comment:
Assuming this refers to the damaging shulker part - if I moved it to the main clock the shulkers would be killed off 4x slower, which I wanted to avoid

Reply:
You can reduce the score before the kill by roughly 1/4th to keep the same duration

Reply:
Oh, of course - for some reason I was thinking the damaging system was something different :/